src/vs/workbench/parts/search/browser/openAnythingHandler.ts

```diff
@@ -76,6 +86,8 @@ export class OpenAnythingHandler extends QuickOpenHandler {
 	}
 
 	public getResults(searchValue: string): TPromise<QuickOpenModel> {
+		const timerEvent = this.telemetryService.timedPublicLog('openAnything');
+		const startTime = timerEvent.startTime ? timerEvent.startTime.getTime() : Date.now();
```

Comment:
I find the check weird here, why would startTime be undefined if you got timerEvent from the timedPublicLog method?